packages/url-parser-node/src/index.ts

```diff
@@ -1,9 +1,8 @@
 import { parseQueryString } from "@aws-sdk/querystring-parser";
 import { Endpoint, QueryParameterBag, UrlParser } from "@aws-sdk/types";
-import { parse } from "url";
```

Comment:
Do you need to`import { URL } from "url";`?

Reply:
No, URL is a Global Object since Node.js v10.0.0
Docs: https://nodejs.org/api/url.html#url_the_whatwg_url_api